enterprise_access/apps/content_assignments/api.py

```diff
@@ -502,6 +569,7 @@ def _create_new_assignments(
         assignment = LearnerContentAssignment(
             assignment_configuration=assignment_configuration,
             learner_email=learner_email,
+            lms_user_id=lms_user_ids_by_email.get(learner_email.lower()),
```

Comment:
Since we pre-computed this, we can easily reuse it when creating new assignments.